```
[2026-02-13T12:27:55.302Z] C:\PROGRA~1\MICROS~2\2022\COMMUN~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe  /nologo /TP -DEXPERIMENTAL_KEY_INSTRUCTIONS -DGTEST_HAS_RTTI=0 -DLLDB_ENABLE_SWIFT -DLLVM_BUILD_STATIC -DUNICODE -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_NONSTDC_NO_WARNINGS -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_ENABLE_EXTENDED_ALIGNED_STORAGE -D_GLIBCXX_ASSERTIONS -D_HAS_EXCEPTIONS=0 -D_SCL_SECURE_NO_DEPRECATE -D_SCL_SECURE_NO_WARNINGS -D_UNICODE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -IT:\5\tools\lldb\tools\lldb-dap\tool -IC:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\lldb\tools\lldb-dap\tool -IT:\5\tools\lldb\tools\lldb-dap -IC:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\lldb\include -IT:\5\tools\lldb\include -IT:\5\include -IC:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\llvm\include -IT:\PythonAMD64-3.10.1\tools\include -IC:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\llvm\..\clang\include -IT:\5\tools\lldb\..\clang\include -IC:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\swift\include -IC:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\swift\stdlib\public\SwiftShims -IT:\5\tools\lldb\..\swift\include -IC:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\lldb\tools\lldb-dap /GS- /Gw /Gy /Oy /Oi /Zc:preprocessor /Zc:inline /Zc:__cplusplus /Zc:inline /Zc:preprocessor /Zc:__cplusplus /Oi /bigobj /permissive- /W4 -wd4141 -wd4146 -wd4244 -wd4267 -wd4291 -wd4351 -wd4456 -wd4457 -wd4458 -wd4459 -wd4503 -wd4624 -wd4722 -wd4100 -wd4127 -wd4512 -wd4505 -wd4610 -wd4510 -wd4702 -wd4245 -wd4706 -wd4310 -wd4701 -wd4703 -wd4389 -wd4611 -wd4805 -wd4204 -wd4577 -wd4091 -wd4592 -wd4319 -wd4709 -wd5105 -wd4324 -wd4251 -wd4275 -w14062 -we4238 /Gw /O2 /Ob2  -std:c++17 -MD   -wd4018 -wd4068 -wd4150 -wd4201 -wd4251 -wd4521 -wd4530 -wd4589  /EHs-c- /GR- -UNDEBUG /showIncludes /Fotools\lldb\tools\lldb-dap\tool\CMakeFiles\lldb-dap.dir\lldb-dap.cpp.obj /Fdtools\lldb\tools\lldb-dap\tool\CMakeFiles\lldb-dap.dir\ /FS -c C:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\lldb\tools\lldb-dap\tool\lldb-dap.cpp
[2026-02-13T12:27:55.302Z] C:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\lldb\tools\lldb-dap\tool\lldb-dap.cpp(382): error C3083: 'WithColor': the symbol to the left of a '::' must be a type
[2026-02-13T12:27:55.302Z] C:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\lldb\tools\lldb-dap\tool\lldb-dap.cpp(382): error C2039: 'error': is not a member of 'llvm'
[2026-02-13T12:27:55.302Z] C:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\llvm\include\llvm/Support/Signals.h(34): note: see declaration of 'llvm'
[2026-02-13T12:27:55.557Z] C:\Users\swift-ci\jenkins\workspace\apple-llvm-project-pull-request-windows\llvm-project\lldb\tools\lldb-dap\tool\lldb-dap.cpp(382): error C2064: term does not evaluate to a function taking 0 arguments
```

Not sure why this is failing to compile, pasting the log here for later.

--- a/lldb/tools/lldb-dap/tool/lldb-dap.cpp
+++ b/lldb/tools/lldb-dap/tool/lldb-dap.cpp
@@ -39,6 +39,7 @@
 #include "llvm/Support/PrettyStackTrace.h"
 #include "llvm/Support/Signals.h"
 #include "llvm/Support/Threading.h"
+#include "llvm/Support/WithColor.h"
 #include "llvm/Support/raw_ostream.h"
 #include <condition_variable>
 #include <cstdio>

